content/_posts/2016-10-28-release-3.2.10.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+date: "2016-10-28"
+title: First Release
+description: details of the first release of Rya (Incubating)
+version: 3.2.10
+fullVersion: rya-incubating-3.2.10
+filename: rya-project-3.2.10-incubating-source-release
+githubRelease: https://github.com/apache/incubator-rya/releases/tag/rel%2Frya-incubating-3.2.10
+apacheRelease: https://dist.apache.org/repos/dist/release/incubator/rya/rya-incubating-3.2.10/
```

Comment:
I had used a similar link on the download page, originally included with the template.  They seem to the same except your link asks before redirecting to the mirror, where the template just redirects.  I'll change to yours in both download page and the release post.